app/controllers/users/verify_account_controller.rb

```diff
@@ -42,5 +43,10 @@ def decrypted_pii
         cacher.fetch
       end
     end
+
+    def next_step
+      return after_sign_in_path_for(current_user) if user_session['openid_auth_request'].present?
+      sign_up_completed_url
```

Comment:
@monfresh the openid connect flow takes the user to a different page after they verify their code, which is determined by the `after_sign_in_path_for` method. Ideally I'd like to just use that everywhere, but I don't want to refactor that method in the context of this PR.

It may also be that we need to update this flow, since we now have different styles for the 'shared attributes' pages. This flow also bypasses the completions page.

For reference, here is a screenshot from `openid_connect` spec on line 280:

![screen shot 2017-05-17 at 9 23 30 am](https://cloud.githubusercontent.com/assets/1421848/26155981/dce9f6fa-3ae2-11e7-8d65-1c4408925f78.png)